src/app/loginComponents/requests/requests.component.html

```diff
@@ -30,8 +30,16 @@ <h3>Curator Requests</h3>
         <mat-card-content class="px-4">
           {{ org.topic }}
         </mat-card-content>
-
         <mat-card-actions fxLayout="row" fxLayoutAlign="end center" fxLayoutGap="10px">
+          <button
+            [id]="'delete-pending-org' + i"
+            mat-raised-button
+            color="warn"
+            [matTooltip]="'Delete this pending organization: ' + org.displayName"
```

Comment:
Not wild about "this" nor the colon. Would just say "Delete pending organization <display name>." Optional.

Reply:
Done.